src/components/Search.vue

```diff
@@ -16,6 +16,7 @@
     <a @click="clearInput">
       <Icon v-if="modelValue" name="close" size="12" class="mb-1" />
     </a>
+    <slot name="searchWithFilters"></slot>
```

Comment:
What's the need of a slot here, can't you just make a SearchWithFilters component like this:
```
<template>
  <Search/>
  <UiDropdown
    class="search-dropdown text-left"
    @redirect="redirectSearch"
    :items="searchOptions"
  >
    <span v-text="searchSelectedOption" />
    <Icon name="arrow-down" class="ml-1" />
  </UiDropdown>
</template>
```

Reply:
@bonustrack already added the SearchWithFilters component but without a slot, dropdown won't be placed inside search bar,

even tried managing with CSS to make it, go inside search bar but clear icon (`X`) doesn't fit well

Reply:
You could manage position relative to the UiButton

Reply:
without slot how can we put dropdown inside `Search` field? 🤔

Reply:
Like this, we dont need the dropdown in the Search component, Search component should be very generic.
```<UiButton>
  <Search/>
  <UiDropdown
    class="search-dropdown text-left"
    @redirect="redirectSearch"
    :items="searchOptions"
  >
    <span v-text="searchSelectedOption" />
    <Icon name="arrow-down" class="ml-1" />
  </UiDropdown>
</UiButton>
```

Reply:
@bonustrack I tried this but to display dropdown inside input field it seems it is impossible without slot :/ (especially with close icon to clear input)

Reply:
@samuveth Any idea on how to do that without having to change Search.vue component? I'd like to keep this component more generic.